static/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -218,6 +220,8 @@ class ConfigureIntegration extends AsyncView<Props, State> {
     const tabs = [
       ['repos', t('Repositories')],
       ['codeMappings', t('Code Mappings')],
+      ['userMappings', t('User Mappings')],
```

Comment:
I think this is going to show the user mappings and team mappings to people who have stack trace linking but may not have codeowners right? seems like we need an extra check here